Python/other/password_genretor.py

```diff
@@ -0,0 +1,33 @@
+import string
+import random
+if __name__ == '__main__':
+    # The lower case for password
+    accsi_1 = string.ascii_lowercase
+
+    #the upper case for password
+    accsi_2 = string.ascii_uppercase
+
+    #The numbers for password
+    digit_1 = string.digits
+    # The symbols for password
+    symbol = ('@,#,$,&,*')          # you can use punctuions function here for the symbols.
```

Comment:
It would be better if you use ```string.punctuation``` as a password can also contain symbols other than the ones you are currently using.